app/reader/filters.py

```diff
@@ -5,8 +5,9 @@
 @reader_bp.app_template_filter('display_date')
 def display_date(dt):
     diff = datetime.now() - dt
+    print(diff)
 
-    if diff.days == 1:
+    if diff.days <= 1:
         return "Today"
     else:
         #return f"{dt.month} {dt.day}, {dt.year}"
```

Comment:
I've heard the community rave about fstrings, why did you change your mind?

Reply:
I just realized now that it should not be commented. I wanted to display "today" for recent articles but the actual date for all others. My bad.

Reply:
cool cool, this isn't related to this comment, but I wonder what the difference between the "Start a Review" and conversation is.

Reply:
Not sure. It would probably contribute to the Code Review stats though.